<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>606</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cascadia Code</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <stylestrategy>PreferAntialias</stylestrategy>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>CRUD</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>931</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="queryLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Query SQL</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>931</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="createBtn">
       <property name="text">
        <string>ADD +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="claveLabel">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Inserte clave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descripcionLabel">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Nombre del producto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="existenciaLabel">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Existencia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="precioLabel">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Precio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>162</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="readBtn">
       <property name="text">
        <string>Ver productos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateBtn">
       <property name="text">
        <string>Actualizar producto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteBtn">
       <property name="text">
        <string>Borrar producto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono</family>
    </font>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
